### DISCUSSION TITLE
I would like to ask where to obtain metadata such as service and endpoint (ID name)

### DISCUSSION THREAD

**Discussion by liuhao710118** [apache/skywalking#12573] [Q&A]
I want to directly obtain data from Elasticsearch, and according to the official documentation, I have correctly obtained metadata such as indicators. But the related services and interfaces in the indicator metadata are all associated in the form of IDs. I would like to ask where to obtain the metadata of services and interfaces, such as the corresponding service ID and service details

![image](https://github.com/user-attachments/assets/dbc27c0c-c0c6-4301-9205-528b3583739e)

**Comment by wu-sheng** [comment]
We don't have suggestions for you. We don't do this.

**Reply by liuhao710118** [reply]
Okay, thank you for your reply

**Comment by liuhao710118** [comment]
The image above is marked incorrectly, it is a serviceId
![image](https://github.com/user-attachments/assets/830bf993-3f15-4d55-a108-a0af2d2f4b76)
